tests/nativeapp/test_post_deploy_for_app.py

```diff
@@ -102,69 +111,15 @@ def test_sql_scripts(
         [
             mock.call(f"use database {app_model.fqn.name}"),
             mock.call(f"use database {app_model.fqn.name}"),
-        ]
+        ],
+        any_order=True,
```

Comment:
There are a bunch of calls in between the two, since sql facade does the whole use_x_optional, so without this the test fails since it expects them in sequence

Reply:
ah fair enough

Reply:
we shouldn't do this. The whole point of the facade is to encapsulate SQL. The strategy we're using is 1) make sure the facade contract is 100% tested by checking the SQL that's being executed. From everywhere else, mock the facade only, not the SQL. If the call is correct, and the facade implementation is right, then we have confidence.

Reply:
I did have a feeling I was missing one more thing I could do to simplify the tests, couldn't quite put my finger on it but that's exactly it, thanks for the pointing that out, just went through all the tests and replaced all the sql executor mocks with sql facade ones, PTAL!